src/hdinsightonaks/azext_hdinsightonaks/tests/latest/test_cluster_with_catalog.py

```diff
@@ -0,0 +1,57 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+import os
+from .testUtil import authorization_info
+from azure.cli.testsdk import ScenarioTest, ResourceGroupPreparer
+
+TEST_DIR = os.path.abspath(os.path.join(os.path.abspath(__file__), '..'))
+
+
+class TestCreateClusterWithCataLog(ScenarioTest):
+    location = 'westus3'
+    resourceGroup = "hilocli-test"
+    clusterPoolName = "hilopool"
+
+    # Set your hive catalog config.
+    secretName = "sqlpassword"
+    catalogName = "hilocli"
+    metastoreDbConnectionURL = "jdbc:sqlserver://yuchensqlserver.database.windows.net:1433;database=clidata;encrypt=true;trustServerCertificate=true;loginTimeout=30;"
+    metastoreDbUserName = "hdi"
+    metastoreDbPasswordSecret = secretName
+    metastoreWarehouseDir = "abfs://clitesta@flinkdemo125stuoi.dfs.core.windows.net/"
+    keyVaultResourceId = "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/Group/providers/Microsoft.KeyVault/vaults/yuchensqlpass"
+
+    # @ResourceGroupPreparer(name_prefix='hilocli-', location=location, random_name_length=12)
```

Comment:
It would be good to add some explanatory comments for this commented-out code, or remove them before they can be tested.

Reply:
Added comments.